frontend/web/components/pages/HomeAside.tsx

```diff
@@ -31,9 +30,29 @@ type HomeAsideType = {
 }
 
 type OptionProps = ComponentProps<typeof components.Option>
+type SingleValueProps = ComponentProps<typeof components.SingleValue>
 type EnvSelectOptionProps = OptionProps & {
   hasWarning?: boolean
 }
+type EnvSingleValueProps = SingleValueProps & {
+  hasWarning?: boolean
+}
```

Comment:
I'm a little unsure on the namings here. What is the difference between `EnvSelectOptionProps` and `EnvSingleValueProps` ?